libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/testbase/testDsl.kt

```diff
@@ -140,7 +141,12 @@ fun TestProject.build(
         enableBuildCacheDebug,
         enableBuildScan,
         gradleVersion
-    )
+    ) + listOfNotNull(kotlinDaemonDebugPort?.let {
+        // Note that we pass "server=n", meaning that we'll need to let the debugger start listening at this port first before the
+        // Kotlin daemon is launched. That is usually easier than trying to attach the debugger when the Kotlin daemon is launched
+        // (currently if we don't attach fast enough, the Kotlin daemon will fail to launch).
+        "-Dkotlin.daemon.jvm.options=-agentlib:jdwp=transport=dt_socket\\,server=n\\,suspend=y\\,address=$it"
```

Comment:
Please change `-Dkotlin.daemon.jvm.options` to `-Pkotlin.daemon.jvmargs`. First one eventually could be deprecated.

Reply:
I see - done